### ISSUE TITLE
Available MLST scheme list is out of date

### ISSUE THREAD

**Issue by eam12** [phac-nml/staramr#168]
The list of MLST schemes available ([https://github.com/tseemann/mlst/blob/master/db/scheme_species_map.tab](https://github.com/tseemann/mlst/blob/master/db/scheme_species_map.tab)) is out of date and so does not work with the `--mlst-scheme` flag for some scheme names. For example, `senterica` is now `senterica_achtman_2`.

**Comment by apetkau** [comment]
Hello @eam12 . Thanks for reporting. Which version of the `mlst` software do you have installed? As in `mlst --version`?

The `--mlst-scheme` option in StarAMR is passed directly to the `mlst` software, as seen below:

https://github.com/phac-nml/staramr/blob/d050a21694f42a33045c5b03c7a626bf0810bcdf/staramr/blast/JobHandler.py#L145-L148

So the valid schemes to use depends on which version of `mlst` you have installed.

**Comment by eam12** [comment]
I have `mlst` version 2.23.0 installed. If I specify the updated scheme name (e.g. senterica_achtman_2) it runs perfectly, but the `--help` menu tells users to look at `mlst`'s scheme_species_map.tab file to get a list of available scheme names. This file is out of date (e.g. still lists `senterica`, not  `senterica_achtman_2 `. It would be great if the `--help` menu directed users to an updated list of `mlst` scheme names. Maybe [https://github.com/tseemann/mlst/blob/master/db/pubmlst](https://github.com/tseemann/mlst/blob/master/db/pubmlst)?

```
--mlst-scheme MLST_SCHEME
                        Specify scheme name, visit https://github.com/tseemann/mlst/blob/master/db/scheme_species_map.tab for supported scheme genus available. [None]
```

**Comment by apetkau** [comment]
Ah, okay. Thanks. Yes, maybe changing the help documentation to point to a different location to get a list of schemes would be helpful (note by default schemes should be auto-detected if you don't specify anything).

**Comment by apetkau** [comment]
Fixed in #186
